src/main/java/org/jenkinsci/plugins/p4/workspace/WorkspaceSpec.java

```diff
@@ -106,7 +114,7 @@ public WorkspaceSpec(String view, String changeView) {
 	public WorkspaceSpec(boolean allwrite, boolean clobber, boolean compress,
 	                     boolean locked, boolean modtime, boolean rmdir, String streamName,
 	                     String line, String view) {
-		this(allwrite, clobber, compress, locked, modtime, rmdir, streamName, line, view, null, null, null, true);
+		this(allwrite, clobber, compress, locked, modtime, rmdir, streamName, "",line, view, null, null, null, true);
```

Comment:
Might need to preserve the Deprecated method and create a new Deprecated method signature (to allow for all the old versions of pipeline code)

Reply:
@p4paul , streamAtChange is added to '@DataBoundConstructor' constructor. This deprecated method calls this constructor, hence adding empty streamAtChange is required. Also, the constructor is used in the Test cases, these test require this change.